### PR TITLE
[Meta DAG] Set time to UTC+2

### PR DESCRIPTION
**Pull request by Pierlou**

### CODE REVIEW

File: meta/task_functions.py [hunk 1]
```diff
@@ -69,19 +69,28 @@ def notification_mattermost(ti):
             hours = int(average_duration // 3600)
             minutes = int((average_duration % 3600) // 60)
             start_time = sorted(successes.keys())[-1].split(' ')[1][:-3]
+            # setting time to UTC+2
+            start_hour = int(start_time.split(':')[0])
```
Comment (maudetes) [inline meta/task_functions.py:73]: Wouldn't it be easier to convert it to datetimes to manipulate instead?

Comment (Pierlou) [reply meta/task_functions.py:73]: Idk if it's easier, I guess more elegant indeed

Comment (maudetes) [reply meta/task_functions.py:73]: less error-prone I would say

**Review by maudetes — APPROVED** [review]
Nice! Looks good to me!

**Pull request merged**